supervisor/shared/rgb_led_status.c

```diff
@@ -118,7 +118,7 @@ void new_status_color(uint32_t rgb) {
         return;
     }
     uint32_t rgb_adjusted = color_brightness(rgb, rgb_status_brightness);
-    current_status_color = rgb_adjusted;
+    current_status_color = rgb;
```

Comment:
I chased `rgb_adjusted` countless times and walked over this every single time. Great catch!!